### PR TITLE
Added Pagination, Sort  and Swagger

### PR DESCRIPTION
**Pull request by MilianCode**

### CODE REVIEW

File: src/main/java/com/miliancode/controller/BookController.java [hunk 1]
```diff
@@ -21,8 +22,8 @@ public class BookController {
     private final BookService bookService;
```
Comment (Elena-Bruyako) [inline src/main/java/com/miliancode/controller/BookController.java:31]: Where is swagger annotations under controller and endpoints?

File: src/main/java/com/miliancode/controller/BookController.java [hunk 2]
```diff
@@ -21,8 +22,8 @@ public class BookController {
     private final BookService bookService;
 
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    public List<BookDto> getAll(Pageable pageable) {
```
Comment (Elena-Bruyako) [inline src/main/java/com/miliancode/controller/BookController.java]:
```suggestion
    public List<BookDto> getAll(@ParameterObject @PageableDefault Pageable pageable) {

```

File: src/main/java/com/miliancode/controller/BookController.java [hunk 3]
```diff
@@ -21,8 +22,8 @@ public class BookController {
     private final BookService bookService;
 
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    public List<BookDto> getAll(Pageable pageable) {
+        return bookService.findAll(pageable);
```
Comment (Elena-Bruyako) [inline src/main/java/com/miliancode/controller/BookController.java:37]: Add validation for arguments where is needed:
```
@RequestBody @Valid CreateBookRequestDto book
@PathVariable @Positive Long id
@Validated under the controller
```

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
Task didn't resolve completely

**Review by okuzan — APPROVED** [review]

**Pull request merged**